tests/financial/financial_integration_test.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+from checkout_sdk.financial.financial import FinancialActionsQuery
+from tests.checkout_test_utils import retriable, assert_response
+from tests.payments.payments_test_utils import make_card_payment
+
+
+def test_should_query_financial_actions(default_api, oauth_api):
+    payment = make_card_payment(default_api, amount=100, capture=True)
+    time.sleep(10)
```

Comment:
There is no need for this if you already have a retriable

Reply:
I'm going to change the signature for retriable to this `def retriable(callback, predicate=None, timeout=2, **kwargs)`,  so that we can set a custom value for the timeout between retries. If not, this test keeps failing.